index.html

```diff
@@ -303,6 +300,38 @@ <h2>Contact Me</h2>
       </div>
     </section>
 
+    <section class="project-popup-wrapper hide">
+      <div class="project-popup">
+        <div class="popup-header">
+          <h2 class="popup-name"></h2>
+          <img class="close-project-popup" src="./img/white-close-icon.svg" alt="close project popup image">
+        </div>
+        <div >
+          <div class="works-intro">
+          <span class="work-name popup-work-name"></span>
+          <ul class="popup-position role"></ul>
+        </div>
+        <img class="popup-image" src="" alt="">
+        <div class="popup-body">
+          <p class="popup-description"></p>
+          <div class="popup-body2">
+            <ul class="popup-technologies-container works-technologies"></ul>
+            <div class="popup-link-container">
+              <a href="#" class="popup-link">
+                See live
+                <img src="./img/source-icon.svg" alt="">
+              </a>
+              <a href="#" class="popup-link">
+                See source
+                <img src="./img/live-demo-icon.svg" alt="">
+              </a>
+            </div>
+          </div>
+        </div>
+      </div>
```

Comment:
- [ ] Based on the requirements of this project, this project section should be created dynamically with the information from the project array hence all the HTML of this section was supposed to be created dynamically with JS when the page load. Kindly fix this.